.github/workflows/ci.yaml

```diff
@@ -115,3 +115,55 @@ jobs:
           platform: ${{ matrix.platform }}
           labels: ${{ needs.prepare.outputs.labels }}
           image-tag: ${{ needs.prepare.outputs.version }}
+
+  push:
+    name: Push images
+    runs-on: ubuntu-latest
+    needs: [prepare, build]
+    strategy:
+      matrix:
+        platform: [amd64, arm64]
```

Comment:
why not build for all supported platforms?

Reply:
done